test/stampit-api.js

```diff
@@ -30,7 +30,7 @@ test('incorrect stampit({ refs }) args', function (t) {
   t.same(stampit({ refs: 42 }).fixed.refs, {});
   t.same(stampit({ refs: null }).fixed.refs, {});
   t.same(stampit({ refs: new RegExp() }).fixed.refs, {});
-  t.same(stampit({ refs: [42] }).fixed.refs, {});
+  t.same(stampit({ refs: [42] }).fixed.refs, { 0: 42 });
```

Comment:
Please, change this and all the rest logics in the tests back. Your changes are wrong.